src/components/Books.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import Book from './sub-components/book';
+import AddBook from './sub-components/AddBook';
+
+export default function Books() {
+  const books = useSelector((store) => store.bookshelf);
+  return (
+    <div>
+      {books.books.map((book) => (
+        <Book
+          key={book.id}
+          title={book.title}
+          author={book.author}
+          id={book.id}
+        />
+      ))}
+      <AddBook />
+    </div>
+  );
+}
+
+Book.PropTypes = {
```

Comment:
- [ ] Please rewrite this line it shows the following error in the console:
  ![image](https://user-images.githubusercontent.com/45002654/166830085-5da6cab5-cab4-42e0-96c0-2f03db6414a4.png)

PropTypes should be written starting with a small letter `p`. See the suggestion below:
  
```JavaScript
Book.propTypes = {
```